src/components/navigation-header/index.tsx

```diff
@@ -4,11 +4,18 @@ import HashiCorpLogo from '@hashicorp/mktg-logos/corporate/hashicorp/logomark/wh
 import InlineSvg from '@hashicorp/react-inline-svg'
 import HeaderSearchInput from 'components/header-search-input'
 import ProductSwitcher from 'components/product-switcher'
-// TODO: we'll need a programatic way to get this data when there are more products
-// TODO: currently we only show Waypoint navigation data
+// TODO: we could likely improve the dictionary approach here.
+// TODO: eg, subnavItems could be a NavigationHeader prop,
+// TODO: and we could pass it in at the layout level.
 import waypointData from 'data/waypoint.json'
+import vaultData from 'data/vault.json'
 import s from './style.module.css'
 
+const subnavItemsDict = {
```

Comment:
This doesn't feel great, but feels like a reasonable stopgap as we add content in. Very open to other suggestions here 🙇‍♂️

Reply:
Is this something that's solved with @ashleemboyer's current product context? 🤔

Reply:
Possibly! I think we'd still need the dictionary of data, but would be leveraging product context as a more standard way of getting `currentProductSlug` (rather than `useRouter` as is now).